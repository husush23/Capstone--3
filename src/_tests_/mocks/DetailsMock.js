/*eslint-disable*/
import React from 'react';

function DetailsMock() {
  return (
    <div className='popup'>
      <div className='popup-content'>
        <h3>Pair name</h3>
        <p>Ask:&nbsp; 454</p>
        <p>Bid:&nbsp; 4554</p>
        <p>Open:&nbsp; 566</p>
        <p>Low: &nbsp;5666</p>
        <span className='back'>&#8592;</span>
      </div>
    </div>
  );
}

export default DetailsMock;
