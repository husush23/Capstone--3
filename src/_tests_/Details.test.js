/* eslintdisable */
import React from 'react';
import { render } from '@testing-library/react';
import DetailsMock from './mocks/DetailsMock';

describe('details', () => {
  test('Renders Details correctly', () => {
    render(<DetailsMock />);
  });
  test('renders Navbar correctly', () => {
    const { asFragment } = render(<DetailsMock />);
    expect(asFragment()).toMatchSnapshot();
  });
});
