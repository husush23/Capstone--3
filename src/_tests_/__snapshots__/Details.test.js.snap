// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`details renders Navbar correctly 1`] = `
<DocumentFragment>
  <div
    class="popup"
  >
    <div
      class="popup-content"
    >
      <h3>
        Pair name
      </h3>
      <p>
        Ask:  454
      </p>
      <p>
        Bid:  4554
      </p>
      <p>
        Open:  566
      </p>
      <p>
        Low:  5666
      </p>
      <span
        class="back"
      >
        ←
      </span>
    </div>
  </div>
</DocumentFragment>
`;
